src/js/Home.js

```diff
@@ -9,18 +9,24 @@ import React from "react";
 import Container from 'react-bootstrap/Container';
 import Row from 'react-bootstrap/Row';
 import Col from 'react-bootstrap/Col';
-
 import Country from './Country';
+import HomePageSelector from './HomePageSelctor';
+
 
 const Home = props => {
+
+    const countryName = localStorage.getItem('countryName');
      return(
+        <>
+        {!countryName && <HomePageSelector  history={props.history} show={true} /> }
         <Container className="Home" fluid>
             <Row className="countryHeadline">
                 <Col sm={12}>
-                    <Country countryName="India" />
+                    <Country history={props.history} countryName={ countryName ? countryName : 'India' } />
```

Comment:
We do not history at all.. Handling the countryName value as state variable will do the tricks

Reply:
history was required to route the user back to the / so that the user will be landed to recently country as the home page.